package.json

```diff
@@ -33,6 +34,8 @@
   ],
   "dependencies": {
     "json-stringify-safe": "5.0.1",
+    "just-extend": "^4.0.2",
+    "lodash": "^4.17.11",
```

Comment:
why? just update the `lodash.assign` version, as lodash is bloated with capabilities we don't need